src/pages/transferWritingPage/TransferWriting.tsx

```diff
@@ -2,39 +2,63 @@ import * as S from "./TransferWriting.style";
 import TransferItem from "./transferItem/TransferItem";
 
 import { fetchTransferItems } from "@/apis/fetchTransferItems";
-import { useToastStore } from "@/store/store";
-import { useSuspenseQuery } from "@tanstack/react-query";
+import { useQuery } from "@tanstack/react-query";
 import { AnimatePresence } from "framer-motion";
 import { useEffect } from "react";
+import { fetchUserInfo } from "@apis/fetchUserInfo.ts";
+import NoResult from "@components/noResult/NoResult";
+import { PATH } from "@constants/path";
+import useToastConfig from "@hooks/common/useToastConfig";
 
 const TransferWriting = () => {
-  const { data } = useSuspenseQuery({
-    queryKey: ["TransferItemList"],
+  const { handleToast } = useToastConfig();
+
+  const { data: userData } = useQuery({
+    queryKey: ["UserInfo"],
+    queryFn: fetchUserInfo,
+  });
+
+  const { data: transferData } = useQuery({
+    queryKey: ["TransferItemList", userData?.id],
```

Comment:
transferData를 받으려면 fetchUserInfo가 먼저 선행되어야 하는 상황이니 enabled 옵션을 통해 종속성을 줄 수 있을 것 같습니다.
https://tanstack.com/query/latest/docs/react/guides/dependent-queries
```suggestion
  const { data: userData } = useQuery({
    queryKey: ["UserInfo"],
    queryFn: fetchUserInfo,
  });

  const { data: transferData } = useQuery({
    queryKey: ["TransferItemList", userData?.id],
    ...
    enabled: !!userData?.id
```

Reply:
와우... 예제가 딱 저를 위해 존재하는 것같은데, 정말 레퍼런스를 잘 찾으시네요!!!

+ 예전에 알려주신 useSuspenseQueries로 하려고 했는데 queryFn에 함수 배열 주고 사용하는 부분에 대한 레퍼런스를 찾지 못했었는데 useQuery를 사용하는 좋은 예제를 보여주셔서 감사합니당..⭐️